### PR TITLE
Update frontend to be compatible with PDB2PQR v3.x backend

### LINKED ISSUE #37: Update PDB2PQR form to adapt to PDB2PQR3 options
Lingering task from Electrostatics/apbs-aws#98

- [x] Remove `typemap` option
- [x] Update CLI flag text for `apbs-input`
- [x] Update CLI flag text for `chain`
- [x] Update flag for pKa options to `titration-state-method`
- [x] Uppercase forcefield CLI args
- [x] ~Add popover for "PDB Selection" section~
- [x] Update how we autofill post-PDB2PQR since output JSON goes away (read APBS file for values)

### PR DESCRIPTION
**Pull request by Eo300**
Resolves #37 

**NOTE**: merging these changes will make the frontend incompatible with the backend until the PR Electrostatics/apbs-aws#123 is also merged

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ class App extends Component {
       let show_cli_args = query_args.get('show_cli')
 
       document.title = "Tools | Configure a PDB2PQR job";
-      bcrumb = this.createServiceBreadcrumb(['Tools', 'PDB2PQR Job Configuration'])
+      bcrumb = this.createServiceBreadcrumb(['Tools', 'PDB2PQR (v3.1.0) Job Configuration'])
```
Comment (intendo) [inline src/App.js]: Can the v3.1.0 be retrieved from the environment or at least a configuration file?

**Comment by Eo300** [comment]
@intendo @mmacduff 
Can we hold on merging this to `main` until we can alert site users via banner/alert (see Electrostatics/apbs-web#39)

File: src/App.js (hunk 1 above)
Comment (Eo300) [reply src/App.js]: Updated the code to do so now.  The defaults present `env/.env` are what we're gonna use, but we can always override at deployment

**Review by intendo — APPROVED** [review]
Very nice!

**Review by mmacduff — APPROVED** [review]

**Comment by sobolevnrm** [comment]
This looks like it has been approved by all reviewers.  Can it be merged?

**Comment by intendo** [comment]
@Eo300 wanted us to wait until he got #39 done. I think he is really close to finishing that task.

**Pull request merged**